docs/TuxActionStores.md

```diff
@@ -0,0 +1,107 @@
+#Tux ActionStores
+##Table of Contents
+<ol>
+  <li><a href="#Premise">Premise</a></li>
+  <li><a href="#Implementation">Implementation</a></li>
+    <ol style="list-style-type:upper-alpha">
+      <li><a href="#Requiring-ActionStores">Requiring ActionStores</a></li>
+      <li><a href="#Creating-Action-Stores">Creating ActionStores</a>
+      </li>
+      <li><a href="#TuxActionStores-Complete-Example">Tux ActionStores Complete Example</a></li>
+    </ol>
+</ol>
+## <a id="Premise"></a>Premise [#](#Premise)
+>Tux `ActionStores` is a module providing the interface for creating stores that are pre-packaged with a 'register' convenience method for automatically registering the store with actions.
+
+## <a id="Implementation"></a>Implementation [#](#Implementation)
+### <a id="Requiring-ActionStores"></a>Requiring Action Stores [#](#Requiring-Action-Stores)
+
+The interface for creating `ActionStores` is exposed via a constructor function:
+
+```
+  var createActionStore = require('tux/Stores/ActionStores');
+```
+
+### <a id="Creating-Action-Stores"></a>Creating ActionStores [#](#Creating-Action-Stores)
+
+Tux `ActionStores` allow users to register their stores with actions by passing in an object of key-value action properties that will be added to the store being created. If the actions passed in contain a 'register' method, then the `ActionStores` will automatically register all the actions with the new store.
+
+
+#### <a id="storeProps"></a>Parameter - `storeProps` - type: OBJECT - required [#](#storeProps)
+StoreProps is an object containing actions that will encapsulate the store's data, with an optional register method which, if provided, will automatically register all the actions with the newly created store. Lets see what this looks like:
+
+```
+    var messageStoreProperties = {
+      _messages: [],
+
+      onGet: function (data) {
+        this._messages = data.messages;
+        this.emitChange();
+      },
+
+      onCreate: function (message) {
+        this._messages.push(message);
+        this.emitChange();
+      },
+
+      onDelete: function (message) {
+        this._messages.splice(message.id, 1);
+        this.emitChange();
+      },
+
+      register: function () {
+        return {
+          get: this.onGet,
+          create: this.onCreate,
+          delete: this.onDelete
+        }
+      }
+    });
+
+```
+
+#### <a id="createActionStore"></a>Method - `actionCategory.register` - type: FUNCTION [#](#createActionStore)
+Creating an `ActionStore` simply requires invoking the createActionStore constructor function:
```

Comment:
This seems like the wrong description.  This description should talk about what the object should look like as well as the fact that function will have the store as its `this` context when it is invoked.  Also mention that this method is essentially a wrapper around `Actions.register` and it submits that store as the first argument and this object as the second argument.  Also mention that this syntax allows you to register listeners with one or multiple actions and one or multiple categories simultaneously:
```
{
  CATEGORY1: {
    CATEGORY1_ACTION1: this.ONTHISACTION,
    CATEGORY1_ACTION2: this.ONANOTHERACTION,
  },
  CATEGORY2: {
    ACTION1: this.ONFINALACTION
  }
}
```